src/stdio.jl

```diff
@@ -52,6 +102,18 @@ function watch_stream(rd::IO, name::AbstractString)
     end
 end
 
+send_stdout(t::Timer) = send_stream("stdout")
+send_stderr(t::Timer) = send_stream("stderr")
+
+function send_stream(name::AbstractString)
+    if bufs[name].size > 0
+        s = takebuf_string(bufs[name])
```

Comment:
bathbytes = convert(Vector{UInt8}, "🛀")
    write(STDOUT, bathbytes[1:2])
    flush(STDOUT)
    sleep(2.0)
    write(STDOUT, bathbytes[3:4])

causes a pretty good error without the change you recommended.